x-pack/plugins/ml/public/application/routing/routes/new_job/recognize.tsx

```diff
@@ -20,7 +20,7 @@ const breadcrumbs = [
   ANOMALY_DETECTION_BREADCRUMB,
   {
     text: i18n.translate('xpack.ml.jobsBreadcrumbs.selectIndexOrSearchLabelRecognize', {
-      defaultMessage: 'Select index or search',
+      defaultMessage: 'Job settings',
```

Comment:
This might be confused with our settings page, our wizards all use something like "Create job / Single metric"
So maybe we could just have "Create jobs" or "Create jobs / Recognized index" or something like that

Reply:
Yes, if this same breadcrumb will be used for all of the steps in the wizard, I think James' suggestion sounds good.